Server/BackEndMonitoringServer/src/CCommandLineHandler.cpp

```diff
@@ -0,0 +1,53 @@
+#include "stdafx.h"
+
+#include "CCommandLineHandler.h"
+
+#include "CServiceHandler.h"
+#include "Utils.h"
+
+Task CommandLineHandler::GetTask()
+{
+    return task;
+}
+
+bool CommandLineHandler::Parse(int argc, char** argv)
+{
+    ServiceParameters parameters;
+    auto service = std::make_unique<CService>(parameters);
+    auto service_handler = std::make_unique<ServiceHandler>(std::move(service));
+
+    bool success = true;
+
+    do
+    {
+        if (argc == 1)
+        {
+            success = service_handler->Run();
+            break;
+        }
+
+	    if (argc == 2)
+        {
+            if (strcmp(argv[1], "install") == 0)
+            {
+                success = service_handler->Install();
+                break;
+            }
+
+            if (strcmp(argv[1], "uninstall") == 0)
+            {
+                success = service_handler->Uninstall();
+                break;
+            }
+
+            if (strcmp(argv[1], "help") == 0)
+            {
+                // Show help
+                break;
+            }
```

Comment:
You could put if -> else if -> else - then if made a mistake, help command will be shown

Reply:
The better option is to change 'if else' on 'switch'

Reply:
Thnx, changed to switch-case